### DISCUSSION TITLE
How do I define the sequence >->> to give ⤖ in equations?

### DISCUSSION THREAD

**Discussion by davidleejy** [typst/typst#3440] [Q&A] [answered]
Hi fellow Typst users,

I'm trying to define a custom shorthand `>->>` to give ⤖ in equations and have yet to figure out how. Sadly, the following do _not_ work:

```typst

#let >->> = $⤖$

#show >->> : ⤖

```

Next, I tried using regex with show. The regex `>-` could match with the text string `">-"` but the regex `>->` could not match with `">->"` as is demonstrated in this brief screen recording:

https://github.com/typst/typst/assets/1690072/730aeb47-d404-4523-a4cd-8bbd980251b7

Would be grateful if someone could share their know-how of how to define a custom shorthand with a string of symbols 😄

**Comment by EpicEricEE** [comment, marked as answer]
You can use the [`quick-maths`](https://github.com/typst/packages/tree/main/packages/preview/quick-maths/0.1.0) package for that:

```typ
#import "@preview/quick-maths:0.1.0": shorthands
#show: shorthands.with(($>->>$, $⤖$))

$ >->> $
```

**Reply by davidleejy** [reply]
Thanks @EpicEricEE, your suggestion works like a charm 🙌🏻
